src/main/kotlin/com/exactpro/th2/lwdataprovider/grpc/GrpcUtils.kt

```diff
@@ -40,7 +43,9 @@ fun List<com.exactpro.th2.dataprovider.grpc.MessageSearchRequest.ResponseFormat>
                 ALL -> list.add(ResponseFormat.ALL)
                 BASE_64 -> list.add(ResponseFormat.BASE_64)
                 PARSED -> list.add(ResponseFormat.PARSED)
-                else -> {}
+                else -> {
+                    logger.error { "Unrecognized response format: $responseFormat" }
+                }
```

Comment:
This is not an error. It is just log message that does not affect anything. We should throw an exception
```suggestion
                else -> error("Unrecognized response format: $responseFormat")
```